Source/Manager.swift

```diff
@@ -70,15 +71,19 @@ public class Manager: NSURLProtocol {
         }
     }
 
+    static public func removeResource(resource: ResourceProtocol) -> ResourceProtocol? {
+        return resources.remove(resource.resourceIdentifier)
```

Comment:
I was looking to emulate the behavior of other remove methods. Like Array's `removeAtIndex(_:)` removes and returns the object instance.